### DISCUSSION TITLE
Why added `order by PRIMARY KEY asc` to `SelectTwoMany`?

### DISCUSSION THREAD

**Discussion by syrflover** [SeaQL/sea-orm#534] [Q&A] [answered]
Hello. thanks for creating great crate this `sea-orm`

I had problem, so  i wrote this post.

[Please check this code.](https://github.com/syrflover/select-two-many-example/blob/master/src/main.rs)

my expected is
```sql
SELECT
  "food_truck"."id" AS "A_id",
  "food_truck"."name" AS "A_name",
  "food"."id" AS "B_id",
  "food"."name" AS "B_name",
  "food"."food_truck_id" AS "B_food_truck_id"
FROM
  "food_truck"
  LEFT JOIN "food" ON "food_truck"."id" = "food"."food_truck_id"
ORDER BY
  "food_truck"."name" DESC
```

but actually
```sql
SELECT
  "food_truck"."id" AS "A_id",
  "food_truck"."name" AS "A_name",
  "food"."id" AS "B_id",
  "food"."name" AS "B_name",
  "food"."food_truck_id" AS "B_food_truck_id"
FROM
  "food_truck"
  LEFT JOIN "food" ON "food_truck"."id" = "food"."food_truck_id"
ORDER BY
  "food_truck"."id" ASC,
  "food_truck"."name" DESC
```

I find this code

https://github.com/SeaQL/sea-orm/blob/e63d4631559c0fac369f0a94fcabebaaf36f1d29/src/query/combine.rs#L130

why added `order by "food_truck"."id" ASC`?

I'm not good english but thanks for read this post

**Comment by billy1624** [comment, marked as answer]
Hey @syrflover, welcome! Good observations! Please find the explanation on https://github.com/SeaQL/sea-orm/issues/424#issuecomment-1024289619

**Comment by syrflover** [comment]
@billy1624, I understand. thanks for comment and have a nice day!

**Comment by billy1624** [comment]
Thanks!
